sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -967,6 +969,27 @@ def dead_letter_source(self):
                 pass
         return None
 
+    @property
+    def message_state(self):
+        # type: () -> Optional[str]
+        """
+        Defaults to Active. Represents the message state of the message. Can be Active, Deferred.
+        or Scheduled.
+
+        :rtype: ~azure.servicebus.ServiceBusMessageState
+        """
+        if self._raw_amqp_message.annotations:
+            try:
+                message_state = self._raw_amqp_message.annotations.get(MESSAGE_STATE_NAME)
+                return (
+                    ServiceBusMessageState.DEFERRED if message_state == 1
+                    else ServiceBusMessageState.SCHEDULED if message_state == 2
+                    else ServiceBusMessageState.ACTIVE
+                )
+            except AttributeError:
+                pass
+        return None
```

Comment:
we check whether the annotations are available and if not, we return `None` which conflicts "Defaults to Active.".

Is this how .NET implement it as well?